nixos/lib/test-driver/test_driver/__init__.py

```diff
@@ -120,7 +129,7 @@ def generate_driver_symbols() -> None:
     in user's test scripts. That list is then used by pyflakes to lint those
     scripts.
     """
-    d = Driver([], [], "", Path())
+    d = Driver([], [], [], "", Path())
```

Comment:
hm we might want to switch to name the parameters to make this less messy. Can you please do it at this occasion?